tests/libs/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 option(ENABLE_DATA_TYPES_TESTS "Enable Data Types tests" TRUE)
+option(ENABLE_GENERATOR_TESTS "Enable block generator tests" TRUE)
```

Comment:
If the lib is called "Block Generator" why not using this name in option naming: `ENABLE_BLOCK_GENERATOR_TESTS`? It seems logical to me to stick with this simple principle instead of letting user to guess the option name.

Reply:
Renamed